packages/flutter_test/test/bindings_environment/flutter_test_variable_is_null_test.dart

```diff
@@ -1,16 +0,0 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-// @dart = 2.9
-
-import 'package:flutter/material.dart';
-import 'package:flutter/widgets.dart';
-import 'package:flutter_test/flutter_test.dart';
-
-void main() {
-  test('$WidgetsBinding initializes with $AutomatedTestWidgetsFlutterBinding when FLUTTER_TEST is defined but null', () {
-    TestWidgetsFlutterBinding.ensureInitialized(<String, String>{'FLUTTER_TEST': null});
```

Comment:
This test is deleted because the type system now grantees that `null` can never be passed in as a value.